code/player/TTTPlayer.Confirmation.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Sandbox;
```

Comment:
File name

`TTTPlayer.PlayerCorpse`? Just nit picking.

Reply:
Thought the same, but `PlayerCorpse.cs` already exists and could lead to confusion.

Reply:
ah right.